### PR TITLE
feat(settings): project followup

### PR DESCRIPTION
**Pull request by MaxBittker**
add empty state and create project buttons

Question: is org:write the correct permissions here?

### CODE REVIEW

**Comment by MaxBittker** [comment]
screenshot 2018-02-02 15 01 47
screenshot 2018-02-02 15 00 54

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by ckj** [comment]
👍 

Fyi, I plan on converting all of the plus signs we're using to actual icons in an upcoming PR.

**Comment by macqueen** [comment]
i did this for the project teams page empty state:

screenshot 2018-02-02 15 18 36

File: src/sentry/static/sentry/app/views/settings/organization/projects/organizationProjectsView.jsx [hunk 1]
```diff
@@ -29,9 +31,29 @@ class OrganizationProjectsView extends OrganizationSettingsView {
 
   renderBody() {
     let {projects} = this.props;
+    let {organization} = this.context;
+    let {access} = organization;
+    let canCreateProjects = access.indexOf('org:write') > -1;
```
Comment (macqueen) [inline src/sentry/static/sentry/app/views/settings/organization/projects/organizationProjectsView.jsx]: hmm i think admins can create new teams and projects but they don't have org write: https://github.com/getsentry/sentry/blob/master/src/sentry/conf/server.py#L1101

Comment (macqueen) [reply src/sentry/static/sentry/app/views/settings/organization/projects/organizationProjectsView.jsx]: maybe `project:admin` ?

File: src/sentry/static/sentry/app/views/settings/organization/projects/organizationProjectsView.jsx [hunk 2]
```diff
@@ -48,6 +70,13 @@ class OrganizationProjectsView extends OrganizationSettingsView {
                 </Box>
               </PanelItem>
             ))}
+            {projects.length === 0 && (
+              <PanelItem align="center">
+                <Box w={1 / 2} p={2} flex="1">
+                  <span>{t('No projects found.')}</span>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/organization/projects/organizationProjectsView.jsx]: `EmptyMessage`

Comment (MaxBittker) [reply src/sentry/static/sentry/app/views/settings/organization/projects/organizationProjectsView.jsx]: oh, nice, thanks

File: src/sentry/static/sentry/app/views/settings/organization/projects/organizationProjectsView.jsx [hunk 3]
```diff
@@ -48,6 +73,13 @@ class OrganizationProjectsView extends OrganizationSettingsView {
                 </Box>
               </PanelItem>
             ))}
+            {projects.length === 0 && (
+              <PanelItem align="center">
+                <Box w={1 / 2} p={2} flex="1">
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/organization/projects/organizationProjectsView.jsx]: nit: un-needed I think?, EmptyMessage has its own padding + centering

**Review by billyvg — APPROVED** [review]

**Pull request merged**